src/platform/packages/shared/kbn-alerting-types/rule_settings.ts

```diff
@@ -35,6 +35,14 @@ export interface RulesSettingsQueryDelayProperties {
 export type RulesSettingsQueryDelay = RulesSettingsQueryDelayProperties &
   RulesSettingsModificationMetadata;
 
+export interface RulesSettingsAlertDeletionProperties {
```

Comment:
Should we do the `...AlertDelete...` renaming here too?

Reply:
Updated in https://github.com/elastic/kibana/pull/216169/commits/472a38d14f13ae3bc22c39c8976e48f83541a05e